lib/ansible/modules/storage/netapp/sf_volume_access_group_manager.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
+
+# (c) 2017, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION = '''
+
+module: sf_volume_access_group_manager
+
+short_description: Manage SolidFire Volume Access Groups
+extends_documentation_fragment:
+    - netapp.solidfire
+version_added: '2.3'
+author: Sumit Kumar (sumit4@netapp.com)
+description:
+- Create, destroy, or update volume access groups on SolidFire
+    - auth_basic
+
+options:
+
+    state:
+        required: true
```

Comment:
That's a good suggestion! However, for the sake of consistency with the other existing NetApp modules, I think it's a good idea to leave it as it is. We can update all of our modules later.